managed/src/main/resources/v1.routes

```diff
@@ -81,11 +81,13 @@ PUT     /customers/:cUUID/releases/:name                                       c
 # Certificate API
 POST    /customers/:cUUID/certificates                                         com.yugabyte.yw.controllers.CertificateController.upload(cUUID: java.util.UUID)
 GET     /customers/:cUUID/certificates                                         com.yugabyte.yw.controllers.CertificateController.list(cUUID: java.util.UUID)
+DELETE  /certificates/:rUUID                                                   com.yugabyte.yw.controllers.CertificateController.delete(rUUID: java.util.UUID)
```

Comment:
/certificates/:rUUID => /customers/:cUUID/certificates/:rUUID

(you should validate the customer with the request too).